dbclient/HiveClient.py

```diff
@@ -300,6 +302,27 @@ def export_hive_metastore(self, cluster_name=None, metastore_dir='metastore/', d
             print("Failed count: " + str(total_failed_entries))
             print("Total Databases attempted export: " + str(len(all_dbs)))
 
+    def get_successful_exported_metastore_tables(self, success_log):
+        tables = set()
+        success_metastore_log_path = self.get_export_dir() + success_log
+        if self.get_checkpoint_enabled() and os.path.exists(success_metastore_log_path):
+            print("Using checkpointing to load completed metastore tables")
+            with open(success_metastore_log_path, 'r') as fp:
+                for table in fp:
+                    item = json.loads(table.rstrip())
+                    tables.add(item['table'])
+        return tables
+
+    def get_successful_imported_metastore_tables(self, success_log):
+        tables = set()
```

Comment:
Similar feedback for the exported table here. We should define whether this is just table name, dbname.tablename, or something else.

Reply:
done